recipe-server/client/control_new/components/extensions/CreateExtensionPage.js

```diff
@@ -0,0 +1,57 @@
+import { message } from 'antd';
+import autobind from 'autobind-decorator';
+import React, { PropTypes as pt } from 'react';
+import { connect } from 'react-redux';
+import { push } from 'redux-little-router';
+
+import ExtensionForm from 'control_new/components/extensions/ExtensionForm';
+import { createExtension } from 'control_new/state/extensions/actions';
+
+
+@autobind
+export class _CreateExtensionPage extends React.Component {
```

Comment:
We haven't used this notation elsewhere. IMO, I don't think we particularly need to denote if the class is private or not, since we have `export`/`export default`.

Reckon this is a good thing for @rehandalal and/or @mythmon to chime in on.